torch/csrc/distributed/rpc/init.cpp

```diff
@@ -693,6 +693,25 @@ PyObject* rpc_init(PyObject* /* unused */) {
       "_disable_server_process_global_profiler",
       &profiler::processglobal::disableServer);
 
+  module.def(
+      "_enable_jit_rref_pickle",
+      &enableJitRRefPickle,
+      R"(
+        Allows ``torch.jit.save`` to pickle a ``torch.jit.ScriptModule`` out of
```

Comment:
seems like we are using torch.jit.save instead of `pickle` to save the ScriptModule, the doc here should be more clear on that I think.

We should probably say:
```
Allows ``torch.jit.save`` to save a ``torch.jit.ScriptModule`` with pickled RRefs out of RPC contexts.
```